databricks/koalas/tests/test_series.py

```diff
@@ -728,3 +728,11 @@ def test_duplicates(self):
 
         self.assert_eq(pser.drop_duplicates().sort_values(),
                        kser.drop_duplicates().sort_values())
+
+    def test_update(self):
+        pser = pd.Series([10, 20, 15, 30, 45], name='x')
+        kser = ks.Series(pser)
```

Comment:
Could you add a test for the case the index is not a simple range? e.g.,

```py
>>> s = pd.Series([1, 2, 3], index=[10, 11, 12])
>>> s
10    1
11    2
12    3
dtype: int64
>>> s.update(pd.Series([4, 5, 6]))
>>> s
10    1
11    2
12    3
dtype: int64
>>> s.update(pd.Series([4, 5, 6], index=[11, 12, 13]))
>>> s
10    1
11    4
12    5
dtype: int64
```

Reply:
okay, i just added them. Thanks for reviewing :)